### DISCUSSION TITLE
[QUESTION] How to export a nova resource to excel WITHOUT actions?

### DISCUSSION THREAD

**Discussion by Vasiliy-Makogon** [SpartnerNL/Laravel-Nova-Excel#150] [General]
I am using Nova and Laravel-Nova-Excel.

I need to export the contents of a nova-resource (+ add any filters) by cron on disk, as Excel. How to do it right?

Now I see this option:

Get the query builder from the resource (how to do it? - the main problem)
Create a query class as described here - https://docs.laravel-excel.com/3.1/exports/from-query.html
Export the file.
Does anyone know other options?

**Comment by arturasfrontit** [comment]
Export from collection
